diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/IdentifierNaming.kt

```diff
@@ -181,11 +181,11 @@ class IdentifierNaming : Rule("identifier-naming") {
      */
     private fun checkObjectNaming(node: ASTNode): List<ASTNode> {
         val objectName: ASTNode? = node.getIdentifierName()
-        // checking object naming, the only extension is "companion" keyword
-        if (!(objectName!!.text.isPascalCase()) && objectName.text != "companion") {
+        // if this object is companion object - it does not have any name
```

Comment:
It corresponds to anonymous objects too.

Reply:
yes, true, but this does not change the logic :) I will fix the comment